design-tokens/build.js

```diff
@@ -298,6 +301,14 @@ function addCoreTokens(tokens) {
   tokens.min = 'min-content';
   tokens.max = 'max-content';
 
+  tokens.px = '1px';
+  tokens.dvw = '100dvw';
+  tokens.dvh = '100dvh';
+  tokens.lvw = '100lvw';
+  tokens.lvh = '100lvh';
+  tokens.svw = '100svw';
+  tokens.svh = '100svh';
```

Comment:
These should be split as well: dvw is for width, while dvh is for height.

Reply:
Looking at tailwind documentation: https://tailwindcss.com/docs/height
They provide both for height and for width, that's why I did the same. I really think we should just be aligned with the default on these things.